src/app/shell/sidenav/sidenav.component.ts

```diff
@@ -11,34 +11,46 @@ import {map, share} from 'rxjs/operators';
   styleUrls: ['./sidenav.component.scss']
 })
 export class SidenavComponent implements OnInit {
+  @Output() sidenavToggle = new EventEmitter();
   userName$: Observable<string>;
   accountMenuItems: MenuItem[];
   sideMenuItems: MenuItem[] = [
     {
       label: 'Dashboard',
       icon: 'pi dashboard-icon',
-      routerLink: ['/app/dashboard']
+      routerLink: ['/app/dashboard'],
+      command: this.onClickSidenav.bind(this)
```

Comment:
te this...bind(this) jakieś brzydkie, nie lepiej po prostu `command: () => this.onClickSidenav()`?
Czy jednak jest to celowe z jakiegoś powodu?
analogicznie niżej

Reply:
a to nie wiedziałem że to tak łyknie, poprawię w takim razie